packages/react-components/semantic-style-hooks-preview/library/src/component-styles/InfoLabel/useSemanticInfoButtonStyles.styles.ts

```diff
@@ -0,0 +1,141 @@
+import { createFocusOutlineStyle } from '@fluentui/react-tabster';
+import { iconFilledClassName, iconRegularClassName } from '@fluentui/react-icons';
+import { makeStyles, mergeClasses, shorthands } from '@griffel/react';
+import { infoButtonClassNames, type InfoButtonState } from '@fluentui/react-infolabel';
+import { getSlotClassNameProp_unstable } from '@fluentui/react-utilities';
+import * as semanticTokens from '@fluentui/semantic-tokens';
+
+/**
+ * Styles for the root slot
+ */
+const useButtonStyles = makeStyles({
+  base: {
+    alignItems: 'center',
+    boxSizing: 'border-box',
+    display: 'inline-flex',
+    justifyContent: 'center',
+    textDecorationLine: 'none',
+    verticalAlign: 'middle',
+    position: 'relative',
+
+    backgroundColor: semanticTokens.nullColor,
+    color: semanticTokens.foregroundCtrlOnTransparentRest,
+
+    ...shorthands.borderStyle('none'),
+    borderRadius: semanticTokens.cornerCtrlRest,
+    margin: '0',
+    padding: semanticTokens.gapInsideCtrlSmDefault,
+
+    [`& .${iconFilledClassName}`]: {
+      display: 'none',
+    },
+    [`& .${iconRegularClassName}`]: {
+      display: 'inline-flex',
+    },
+
+    ':hover': {
+      backgroundColor: semanticTokens.nullColor,
```

Comment:
Need to use _nullBackgroundColorHover here - the hover color ensures that the high contrast color kicks in

Reply:
Actually nvm, button should also use 'colorTransparentBackgroundHover' also and doesn't so going to update and add here